From: Jon Bernard <jbernard@debian.org>
Date: Sun, 27 Mar 2011 21:14:46 -0400
Subject: Update LSB function calls in initscripts
To: libcg-devel@lists.sourceforge.net

In Debian, these functions go by a slightly different name and
signature.
---
 scripts/init.d/cgred.in |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/scripts/init.d/cgred.in b/scripts/init.d/cgred.in
index d0b343b..9b30908 100644
--- a/scripts/init.d/cgred.in
+++ b/scripts/init.d/cgred.in
@@ -81,7 +81,7 @@ start()
 		log_failure_msg $"Cannot find cgroups, is cgconfig service running?"
 		return 1
 	fi
-	daemon --check $servicename --pidfile $pidfile $CGRED_BIN $OPTIONS
+	start_daemon -p $pidfile $CGRED_BIN $OPTIONS
 	retval=$?
 	echo
 	if [ $retval -ne 0 ]; then
@@ -125,7 +125,7 @@ case "$1" in
 		RETVAL=$?
 		;;
 	status)
-		status -p $pidfile $servicename
+		status_of_proc -p $pidfile $servicename
 		RETVAL=$?
 		;;
 	restart)
-- 
